ui/src/features/configurationColumn.js

```diff
@@ -32,7 +33,7 @@ const semiLarge = 70;
 const largeSize = 85;
 const extraLargeSize = 100;
 const extraExLargeSize = 120;
-export const getColumns = ({columnsNames, sortHeader = '', onSort, onReportView, onRawView, onStop, onDelete, onEdit, onRunTest, onEnableDisable, onEditNote, selectedReports, onReportSelected}) => {
+export const getColumns = ({columnsNames, sortHeader = '', onSort, onReportView, onRawView, onStop, onDelete, onEdit, onRunTest, onEnableDisable, onEditNote, selectedReports, onReportSelected, onClone}) => {
```

Comment:
when button is pressed, don't clone the test but open a test editor with test data + concat test name: 'clone'

I.e test name was Hello World
open form with test data and name it Hello World Clone